src/domainobjectfactories/instrument_factory.py

```diff
@@ -132,3 +190,63 @@ def __create_exchange_code(self, random_exchanges_row):
             f"exchange_code not present in row.  Fields are: " \
             f"{random_exchanges_row.keys()}"
         return random_exchanges_row['exchange_code']
+
+    def __create_issuer_name(self):
+        """Create a random 10 character issuer name
+
+        Returns
+        -------
+        String
+            Randomly generate a ten character string to be issuer name
+        """
+        return self.create_random_string(length=10, include_numbers=False)
+
+    @staticmethod
+    def __create_figi():
+        """Create a random valid FIGI
+
+        Returns
+        -------
+        String
+            Randomly generate a FIGI
+        """
+
+        consonants = ['B', 'C', 'D', 'F', 'G', 'H', 'J', 'K', 'L', 'M', 'N',
+                      'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'Y', 'Z']
+        consonants_and_numbers = consonants + ['1', '2', '3', '4', '5', '6',
+                                               '7', '8', '9']
+
+        random.shuffle(consonants)
+        combination_generator = map(''.join,
+                                    itertools.permutations(consonants, 2))
+        invalid_combinations = ['BS', 'BM', 'GG', 'GB', 'GH', 'KY', 'VG']
+
+        while True:
+            combination = next(combination_generator)
+            if combination not in invalid_combinations:
+                break
+
+        character_three = 'G'
+
+        characters_four_to_eleven = \
+            random.choices(consonants_and_numbers, k=8)
+
+        # TODO Currently the final digit is being randomly generated,
+        #  whereas in a true FIGI it is based on the preceding characters
+        character_twelve = str(random.randint(0, 9))
+
+        return combination + character_three + ''.join(
+            characters_four_to_eleven) + character_twelve
+
+    @staticmethod
+    def __get_industry_classification():
+        """Randomly select an industry classification
+
+        Returns
+        -------
+        String
+            A randomly chosen industry classification
+        """
+        industry_classifications = \
+            ['MANUFACTURING', 'TELECOMS', 'FINANCIAL SERVICES', 'GROCERIES']
+        return random.choice(industry_classifications)
```

Comment:
can we move

`industry_classifications = ['MANUFACTURING', 'TELECOMS', 'FINANCIAL SERVICES', 'GROCERIES']`

up to the top of the class definition so that it is a class attribute, in the same manner as `ASSET_CLASS_TO_SUBCLASS`. Also capitalise to keep in the capital letter constant convention.

Then we can change this method to:

```
def __get_industry_classification(self):
    return random.choice(self.INDUSTRY_CLASSIFICATIONS)
```